### ISSUE TITLE
Heroku Deploy Succeeds, but assets 404.

### ISSUE THREAD

**Issue by mattmonihan** [daptin/daptin#72]
- Are you running a release or master: master
- Issue is about a fresh instance (no data in db) or restart: Fresh instance.

I used the "deploy to heroku" button to get up and running, which succeeded, but when I navigate to the app's index page, `/`, I'm getting a blank screen. All the assets are getting loaded except for the following, which are 404ing.

/static/js/app.028c9ad5f86af201da30.js `404`
/static/js/vendor.7608ce94784a3aaaf8ea.js `404`

Is there another piece of configuration necessary?

When the app starts, I'm getting some of these messages:

```
msg="Failed to scan config value [hostname]: sql: no rows in result set"
2020-02-07T22:36:35.348786+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=info msg="Failed to scan config value [jwt.secret]: sql: no rows in result set"
2020-02-07T22:36:35.352596+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=info msg="Failed to scan config value [logs.enable]: sql: no rows in result set"
2020-02-07T22:36:35.356512+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=info msg="Failed to scan config value [graphql.enable]: sql: no rows in result set"
2020-02-07T22:36:35.361962+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=info msg="Failed to scan config value [encryption.secret]: sql: no rows in result set"
2020-02-07T22:36:35.366242+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=info msg="Failed to scan config value [jwt.token.issuer]: sql: no rows in result set"
2020-02-07T22:36:35.366250+00:00 app[web.1]: time="2020-02-07T22:36:35Z" level=error msg="No default jwt token issuer set: sql: no rows in result set"
```

**Comment by artpar** [comment]
Hi,

So heroku buildpack doesn't build the web dashboard when it deploys the go project.

For now I have built it and added the dist folder to git, so heroku deployment will find the files at the expected location now. I will try to keep the web dist folder available from now on in git itself for Heroku. 

You will need to deploy again using the same way to now be able to access the dashboard. I just tried a test deployment and able to see the dashboard now.

Do note that the default heroku installation is running on a sqlite db and should not be used for heavy loads. For production purposes consider using mysql/postgres backed instances.
